lib/engine/game/g_1822_ca/step/destination_token.rb

```diff
@@ -80,6 +81,8 @@ def process_place_token(action)
           end
 
           def destination_node_check?(entity)
+            return @connected_destination_nodes if @dest_checked
```

Comment:
This isn't the same thing normally returned by this function.

Fixed in #10964